circle.yml

```diff
@@ -1,31 +1,30 @@
-# circle.yml -- Font-Awesome
-
 machine:
   environment:
     DEPLOY_TOOLKIT: $HOME/deploy_toolkit
     FAILURE_FILE: /tmp/build_failures.txt
+    DEPLOY_TOOLKIT_BRANCH: <DEPLOY_TOOLKIT_BRANCH>
+  node:
+    version: 6.9.0
+
 dependencies:
   pre:
-    # Need deploy_toolkit before anything else
-    - git clone -b master git@github.com:loanlifecycle/deploy_toolkit.git $DEPLOY_TOOLKIT
-    # These are used by build_notifications.py
-    - pip install paho-mqtt mock nose2 cov-core requests pycurl
-    - sudo apt-get install --assume-yes --force-yes ruby-dev nodejs npm build-essential
+    - |
+      if [ "$DEPLOY_TOOLKIT_BRANCH" == "<DEPLOY_TOOLKIT_BRANCH>" ]; then DEPLOY_TOOLKIT_BRANCH="master"; fi
+      echo "Checking out branch: $DEPLOY_TOOLKIT_BRANCH\n"
+      git clone -b $DEPLOY_TOOLKIT_BRANCH git@github.com:loanlifecycle/deploy_toolkit.git $DEPLOY_TOOLKIT
+    - sudo apt-get install --assume-yes --force-yes ruby-dev npm
```

Comment:
Do we need to specify npm here with node in the machine section? Could we specify ruby in the machine section as well?